VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -188,6 +197,12 @@ def controller_id_and_lun_to_settings_data(scsi_controller, lun_number):
             "KekAlgorithm": kek_algorithm,
             "Disks": data_disks_settings_data
         }
+
+        self.logger.log("Settings without the protectors array: " + json.dumps(data, sort_keys=True, indent=4))
```

Comment:
>without the protectors array [](start = 34, length = 28)

We should dump the protectors Name without the key in the JSON. This will let us know if the protectors section was added with the correct protector name or not.